utils/resource-utils.js

```diff
@@ -105,8 +105,12 @@ export const getResourceDetails = (uri) => {
   );
 
   if (!resourceMatch) {
-    console.error("Unknown resource type found");
-    return null;
+    console.log("Resource URI does not match expected format");
```

Comment:
Was this `console.log` for debugging?

Reply:
I wasn't sure if it would be useful to keep the log around for the Verisign folks since they know they have some URIs that don't match the expected format. I figured eventually this could be logged somewhere more legitimate but I can take it out for now if anyone feels strongly about it.

Reply:
I think it's fine to leave it

Reply:
for now, they'll probably find it very useful